tests/Feature/Api/Administration/UsersTest.php

```diff
@@ -222,6 +222,7 @@ public function testUpdateUser()
             'username' => app()->make('Faker\Generator')->text(10),
             'firstname' => app()->make('Faker\Generator')->text(10),
             'lastname' => app()->make('Faker\Generator')->text(10),
+            'status' => 'INACTIVE'
```

Comment:
This test needs to be expanded. Test that a user is inactive and then verify that you cannot login with it. Just checking for the 200 is not enough.